commands/activations.go

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
These look to have diverged from the code generated by the https://github.com/digitalocean/doctl-sandbox-plugin 

Is that correct?

Reply:
Yes.   I have concluded that we can't keep depending on the codegen, except when importing new bunches of functions.   So, there is a substantial manual editing pass reflected in this PR.

Reply:
I believe we have complete freedom.